README.md

```diff
@@ -217,21 +217,107 @@ Keen.ready(function(){
 Read more about advanced queries in our [query guide](./docs/query.md).
 
 
-## Query Caching
+## Saved Queries
 
-Data sent to Keen is available for querying almost immediately. For use cases that don’t require up-to-the-second answers but require fast performance, query caching can be used to speed up a query. To include query caching as a feature, just add the `maxAge` query parameter to any other query parameters you’ve already specified. The first time your application makes a query specifying the max_age the answer will be calculated normally before it can be cached for future uses.
+There are multiple ways to interact with the saved queries API.
+
+### Example Usage
 
 ```javascript
-var count = new Keen.Query("count", {
-    eventCollection: "pageviews",
-    groupBy: "property",
-    timeframe: "this_7_days",
-    maxAge: 300 // include maxAge as a query parameter to activate Query Caching
+// Create a client instance
+var client = new Keen({
+  projectId: "YOUR_PROJECT_ID",
+  readKey: "YOUR_READ_KEY"
+});
+
+Keen.ready(function() {
+  var savedQuery = this.client.savedQuery();
```

Comment:
is `this.` required here? Seems like it wouldn't be, but maybe I'm missing something in the diff view.